waitList/style.css

```diff
@@ -0,0 +1,26 @@
+.container {
+    width: 30%;
+    display: grid;
+    border: 3px solid goldenrod;
+    border-radius: 10px;
+    margin: 50px auto;
+    height: 480px;
+}
+.test {
+    width: 80%;
+    margin-left: 40px;
+}
+.text {
+    padding: 15px;
+    margin-left: 30px;
+    font-size: 20px;
+    font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
```

Comment:
I will advice we all agree as to what font-family to use for uniformity.
Thank you so much.
This looks nice.

Can we also try handling the responsive view and a snapshot of our work?